.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,58 @@
+name: continuous-integration
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+  release:
+    types: [published]
+
+jobs:
+  continuous-integration:
+    name: continuous-integration
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: install dependencies
+        run: npm install
```

Comment:
Let's use `npm ci` to we can avoid inconsistent behaviour due to new deps versions